crowdsec-docs/unversioned/getting_started/post_installation/health_check.mdx

```diff
@@ -0,0 +1,335 @@
+
+## ✋🏻 Remediation checks
+
+### *Validate Blocks or Captchas* 
+
+Now that detection and connectivity are working, let’s verify that your bouncers are correctly applying remediation on malicious IPs.  
+
+**Prerequisite:**  
+To apply remediation with CrowdSec, you’ll need a bouncer — available for firewalls, web servers (like Nginx), reverse proxies, CDNs, cloud WAFs, edge appliances, [and more](https://doc.crowdsec.net/u/bouncers/intro).  
+
+<details>
+  <summary>✋🏻 Bouncer Remediation test</summary>
+
+This test involves manually creating a **decision** against a public IP of one of your devices for a very short period (1 minute).  
+
+:::danger 
+BE CAREFUL -- Risk of Self-Lockout  
+This procedure will temporarily block your access to the services protected by your bouncer.  
+Make sure to properly follow the instructions to set the TTL to a low expiration time (1 minute). 
+OR do it from a device with a different public IP address than the client you're using to setup CrowdSec.
+:::
+
+1️⃣ Find your public IP:
+<CodeBlock className="language-bash">curl api.ipify.org</CodeBlock>
+or
+<CodeBlock className="language-bash">curl curl ipinfo.io/ip</CodeBlock>
+
+2️⃣ Add a ban decision for your IP (valid for 1 minute):
+<CodeBlock className="language-bash">sudo cscli decisions add ban --ip \<your-public-ip\> --duration 1m --reason "CrowdSec remediation test"</CodeBlock>
+
+3️⃣ Try accessing your service (e.g. website, API). from the same public IP address. 
+  ➡️ You should be blocked by the bouncer. returning a forbidden response (HTTP 403) or a captcha challenge.
+
+4️⃣ Wait for 1 minute, then check the decisions list to see if the decision has been removed
+
+</details>
+
+### Were all the tests successful ?
+
+If you were successfully blocked, congratulations! Your remediation setup is working correctly. 🎉  
+
+You might want to continue to the next recommended steps:
+- Enroll your Security Engine to the [CrowdSec Console](/u/getting_started/post_installation/console/)
+- Then subscribe to more blocklists to benefit from additional proactive prevention
+
+<details>
+    <summary>🐞 **Remediation Troubleshooting**</summary>
+
+  Before diving into troubleshooting, remember that a bouncer is a separate component that connects to the Security Engine and regularly pulls decisions (like bans or captchas) to apply them at its level (firewall, web server, etc.). If remediation isn’t working, it’s often due to issues in this communication loop.  
+
+  <details>
+      <summary>Bouncer Configuration Troubleshooting</summary>
+      
+  *   Verifying that the bouncer is registered and active.
+  *   Checking bouncer logs for errors.
+  *   Troubleshooting communication issues between the bouncer and the CrowdSec LAPI.
+  *   Firewall configuration for the bouncer.
+  *   Specific troubleshooting steps for common bouncers (firewall-bouncer, nginx-bouncer, etc.).)
+
+  </details>
+
+  <details>
+      <summary>Profile Troubleshooting</summary>
```

Comment:
Should this be here ?

If the user comes here directly after the install (depending on how we link to this page), it's very unlikely for them to have modified the profiles configuration.

Maybe this should be part of a troubleshooting section in the profile guide ?